### DISCUSSION TITLE
File version

### DISCUSSION THREAD

**Discussion by pedmondson** [haplokuon/netDxf#289] [Q&A] [answered]
i have DXF files produced in version AC1014.   Is it possible to read these without requiring to upgrade the Dxf to version AC1027 which I am presently having to?

**Comment by haplokuon** [comment, marked as answer]
It is in the README, AC1015 and upwards.

**Reply by VinayFdes** [reply]
Hello,

I am trying to load a dxf but it shows this error: "DXF file version not supported : AutoCad10.". 
Can you please suggest How can I load this file ?

**Comment by pedmondson** [comment]
Open it in notepad and look at the 8th line.  There is will be code that starts with AC  i.e AC1015.   See what is then look it up in this table for required cad version

https://knowledge.autodesk.com/support/autocad/learn-explore/caas/sfdcarticles/sfdcarticles/drawing-version-codes-for-autocad.html
